.gitleaks.toml

```diff
@@ -0,0 +1,5 @@
+[allowlist]
```

Comment:
what is this file for?

Reply:
For posterity, redhat has a scanner that checks content that gets checked in for secrets and it found the RSA key I put into the _test.go file. This file adds an exception for that.